test/jest/gpt.js

```diff
@@ -30,7 +30,14 @@ describe('setup', () => {
 
 	describe('enableLazyLoad()', () => {
 
-		test('should explicitly disable lazy load', () => {
+		test('should implicitly disable lazy load, if option not provided', () => {
```

Comment:
Not a biggie, but I think removing the word 'implicitly', makes it clearer:
 
```suggestion
		test('should disable lazy loading, if the "enableLazyLoad" option is not provided', () => {
```